assignments/prototype-refactor.js

```diff
@@ -4,6 +4,122 @@ Prototype Refactor
 
 1. Copy and paste your code or the solution from yesterday
 
+
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+/* dimensions (These represent the character's size in the video game)
+* destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+
+class GameObject {
+    constructor (attributes) {
+    this.createdAt = attributes.createdAt;
```

Comment:
Does `.createdAt` need `()` at the end since it's a function, so we can invoke it? or does it not? 
If it doesn't, look into why...
If it does than, figure out why as well...
reference that attribute inside of your object itself